packages/chrome/src/elements/sheet/Sheet.tsx

```diff
@@ -51,6 +51,7 @@ const SheetComponent = React.forwardRef<HTMLElement, ISheetProps>(
     return (
       <SheetContext.Provider value={sheetContext}>
         <StyledSheet
+          inert={isOpen ? undefined : ''}
```

Comment:
Adding mental notes that this might break with React@19.x:

https://github.com/facebook/react/issues/17157#issuecomment-2003750544

Reply:
using an empty string feels wrong anyway lol glad they're fixing it.

Reply:
the unit tests should tell us if the fix breaks Garden's code